pkg/controller/controller.go

```diff
@@ -151,11 +158,21 @@ func (c *CSIControllerService) Watch(*grpc_health_v1.HealthCheckRequest, grpc_he
 // For example storageType: HDD, storageType: HDDLVG. If this field is not set then storage type would be ANY.
 // Receives golang context and CSI Spec CreateVolumeRequest
 // Returns CSI Spec CreateVolumeResponse or error if something went wrong
-func (c *CSIControllerService) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest) (*csi.CreateVolumeResponse, error) {
+func (c *CSIControllerService) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest) (retresp *csi.CreateVolumeResponse, reterr error) {
 	ll := c.log.WithFields(logrus.Fields{
 		"method":   "CreateVolume",
 		"volumeID": req.GetName(),
 	})
+	defer c.metricCreateVolume.EvaluateDurationForMethod("CreateVolume", prometheus.Labels{"volume_name": req.Name})()
+	defer func() {
+		if retresp != nil && reterr == nil && ctx.Err() == nil {
+			go func() {
```

Comment:
when will this goroutine exit ?

Reply:
3 minutes later according to DbgMetricHoldTime.